### DISCUSSION TITLE
how do i include -c -d -o arguments/flags to src/r/edger.r if running from source in RStudio?

### DISCUSSION THREAD

**Discussion by njbowen** [biostars/biostar-handbook-issues#274] [Q&A] [answered]
how do i include -c -d -o arguments/flags to src/r/edger.r if running from source in RStudio?

**Comment by ialbert** [comment, marked as answer]
Each script has default values for each parameter at the start, in RStudio one would need to replace those values with the desired values.

**Reply by ialbert** [reply]
See also here:

RNA-Seq book:

https://www.biostarhandbook.com/books/rnaseq/rnaseq-rstudio.html

or for workflows:

https://www.biostarhandbook.com/books/workflows/install/rstudio/#running-scripts-in-rstudio

**Reply by njbowen** [reply]
ok. so I can't change them on the fly for the script within a script, gotcha. thanks.
